packages/edit-post/src/editor.js

```diff
@@ -12,41 +17,61 @@ import { KeyboardShortcuts } from '@wordpress/components';
 import preventEventDiscovery from './prevent-event-discovery';
 import Layout from './components/layout';
 
-function Editor( {
-	settings,
-	hasFixedToolbar,
-	focusMode,
-	post,
-	initialEdits,
-	onError,
-	...props
-} ) {
-	if ( ! post ) {
-		return null;
+class Editor extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.getEditorSettings = memize( this.getEditorSettings, {
+			maxSize: 1,
+		} );
+	}
+
+	getEditorSettings( settings, hasFixedToolbar, focusMode ) {
```

Comment:
This function is never called. I assume it was meant to be used in assigning to the result of `editorSettings` in the `render` function.

Reply:
Good catch, let's remove it.

Reply:
> Good catch, let's remove it.

Remove? Or use? I've updated to use it in #14224, but it may make sense to have a separate pull request anyways.

Reply:
Sorry, yes, let's use it :). I had something else in mind.

It doesn't seem like a regression that needs to land in 5.2, so I don't have a strong opinion about having a separate PR.